src/main/java/ca/on/oicr/pineryreports/reports/impl/LanesBillingReport.java

```diff
@@ -205,15 +205,27 @@ private ReportObject getReportAndSummaryData(Collection<RunDto> runs, Map<String
     for (RunDto run : runs) {
       String instrumentName = getInstrumentName(run.getInstrumentId(), instrumentsById);
       String instrumentModel = getInstrumentModel(run.getInstrumentId(), instrumentsById, instrumentModelsById);
-      if (run.getPositions() == null) continue;; 
+      if (run.getPositions() == null) continue;
       for (RunDtoPosition lane : run.getPositions()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)